samples/net/google_iot_mqtt/sample.yaml

```diff
@@ -1,2 +1,8 @@
+common:
+  harness: net
+  tags: net
```

Comment:
`mqtt` tag could be added here so that it is possible to run just mqtt specific stuff